plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -277,6 +277,7 @@ describe('pg-pool', () => {
         await client.query('SELECT NOW()');
       } finally {
         client.release();
+        await newPool.end();
```

Comment:
This is an unrelated change. It fixes a few-second hang in the test file completing (presumably on a pool connection timeout).